snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc

```diff
@@ -54,3 +57,4 @@ spec:
 <13> Lists the tenants.
 <14> The tenant name from the `X-Scope-OrgId` header when ingesting the data.
 <15> The unique identifier of the tenant. Must be unique throughout the lifecycle of the TempoMonolithic deployment. This ID will be added as a prefix to the objects in the object storage. You can reuse the value of the UUID or `tempoName` field.
+<16> Optional: Accepts arbitrary configuration that might not be supported by Red Hat. Red Hat supports only documented first-class custom resource options.
```

Comment:
> that might not be

shall we be more explicit and say it's not supported? 

cc) @andreasgerstmayr ?